### DISCUSSION TITLE
Cout not giving output

### DISCUSSION THREAD

**Discussion by KeshavGupta-0** [microsoft/vscode-cpptools#12811] [Q&A] [answered]
Hello everyone,
I am trying to learn c++ but cout and cin not working. Cout is giving just a blank line. I tried printf and it is working. I have installed compiler also.I have tried std::cout and std::endl together but it doesn't work too. Please help me to solve this problem.

**Comment by bobbrow** [comment]
Can you share your code and tell which which compiler (and version) you are using?  Please also share which OS you're using.

**Reply by KeshavGupta-0** [reply, marked as answer]
Thank you for replying. My problem is solved now. It turned out that my GCC compiler was outdated.
